tekdoc/tekdoc_test.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"bytes"
+	"github.com/google/go-cmp/cmp"
+	"testing"
+)
+
+func TestTekdoc(t *testing.T) {
+	task, err := read("buildkit-daemonless.yaml")
+	if err != nil {
+		t.Fatal(err)
+	}
+	if task.Kind != "Task"{
+		t.Error("This is not a task file")
```

Comment:
We should include the `Kind` we got in the error message so it's easier to debug if/when this fails.